plugins/btrfs-subvolumes-mounted.sh

```diff
@@ -1,10 +1,10 @@
 #!/bin/bash
 
 run_checks() {
-
-    MOUNTS=`grep "btrfs.*subvol=" /etc/fstab | awk '{print $2}' | sed -e 's|^/||g' -e 's|-|\\\x2d|g' -e 's|\.|\\\x2e|g' -e 's|/|-|g'`
+    MOUNTS=`grep "btrfs.*subvol=" /etc/fstab | awk '{print $2}'`
     for i in ${MOUNTS}; do
-        systemctl is-failed -q $i.mount
+        path=$(systemd-escape -p "${i}")
```

Comment:
Thanks a lot! Thinking of this, there is still one case where it may fail with the current code: If _/etc/fstab_ contains octal encoded strings (such as a space: `\040`), then the encoding will still be different from systemd, which encodes this as `\x20`.

So if `${i}` was replaced with `$(echo -e ${i)}` to first decode the string I think it should really catch all combinations.